### DISCUSSION TITLE
Unexpected space between segments/text

### DISCUSSION THREAD

**Discussion by comanche** [JanDeDobbeleer/oh-my-posh#668] [Troubleshoot] [answered]
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] I looked for duplicate issues before submitting this one

### Description

After running a command, the prompt display a space between the **executiontime** segment and the **exit** segment.

### Environment

- Oh my Posh version: 3.139.2
- Theme: jandedobbeleer
- Operating System: Windows 10 64-bit 20H2 19042.928
- Shell: PowerShell 7.1.3
- Terminal: Windows Terminal 1.7.1033.0

### Steps to Reproduce

1. Install Windows Terminal on a clean Windows 10 64-bit 20H2 19042.928 installation
2. Install & import oh-my-posh
3. Run `Set-PoshPrompt -Theme jandedobbeleer`
4. Run any command (e.g. gci).

**Expected behavior:** [What you expected to happen]

There is no space between the **executiontime** segment and the **exit** segment.

**Actual behavior:** [What actually happened]

There is a black space between the **executiontime** segment and the **exit** segment.

![image](https://user-images.githubusercontent.com/1100167/115304354-7d6e2180-a12a-11eb-80a3-a02c589143cf.png)

**Comment by lnu** [comment]
Same issue here but it's a windows terminal issue. Related to this one I think https://github.com/microsoft/terminal/issues/3546.

There are still lots of issues related to unicode width: https://github.com/microsoft/terminal/issues?q=is%3Aissue+is%3Aopen+unicode+width

**Comment by gitolicious** [comment, marked as answer]
Not exactly sure why, but what helped for me was adding an invisible `\u2800` character at the end.
```jsonc
        {
          "type": "executiontime",
          /* other attributes here */
          "properties": {
            "always_enabled": true,
            "prefix": "\ufa1e",
            "postfix": "\u2800" // invisible spacing character
          }
        },
```
It's an ugly workaround but the result looks nice.
![image](https://user-images.githubusercontent.com/26963495/115928894-28624080-a487-11eb-8b78-cdf7ff2f59a0.png)

**Reply by comanche** [reply]
@gitolicious Thanks for the temporary fix.  That worked for me.

**Reply by codeRSH** [reply]
This Worked. Thanks!
